.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2.1
```

Comment:
It doesn't look like I have access to actually turn on circleci for this repo, but I think it would be valuable.